src/components/new-members/index.js

```diff
@@ -16,19 +17,18 @@ const renderNewUserCard = (newMember) => {
   );
 };
 
-const renderNewUser = (newMember, isSuperUser) => {
+const renderNewUser = (newMember, isSuperUser, handleNewMemberClick) => {
   if (isSuperUser) {
     return (
-      <Link
-        prefetch={false}
-        href={{
-          pathname: '/[id]',
-        }}
-        as={`/${newMember.username}`}
+      <div
+        role="button"
+        tabIndex={-1}
```

Comment:
tabIndex needs to be a non negative number for the tab to focus on the element since aria-hidden is set to true